nemo/collections/asr/models/ctc_models.py

```diff
@@ -544,27 +551,32 @@ def training_step(self, batch, batch_nb):
         else:
             log_probs, encoded_len, predictions = self.forward(input_signal=signal, input_signal_length=signal_len)
 
+        if hasattr(self, '_trainer') and self._trainer is not None:
+            log_every_n_steps = self._trainer.log_every_n_steps
+        else:
+            log_every_n_steps = 1
+
         loss_value = self.loss(
             log_probs=log_probs, targets=transcript, input_lengths=encoded_len, target_lengths=transcript_len
         )
 
         # Add auxiliary losses, if registered
         loss_value = self.add_auxiliary_losses(loss_value)
+        loss_value, tensorboard_logs = self.add_interctc_losses(
```

Comment:
Very cool. Can you add a comment why we have the odd bool check in the `compute_wer` part.